### DISCUSSION TITLE
401 (Unauthorized) from remark.mjs

### DISCUSSION THREAD

**Discussion by jhfoo** [umputun/remark42#1251] [Q&A]
Tried the sample code to start the service. Made some changes to get it to work:
```
    <script>
      window.remark_config = {
        host: 'http://192.168.86.178:8080',
        site_id: 'remark',
      }
    </script>
```

Notice `window.` above.

So the UI loaded but on the browser console output I see this:
```
 GET http://192.168.86.178:8080/api/v1/user?site=remark 401 (Unauthorized)            remark.mjs:2
```

Is this to do with the secret I entered to start the server? Not sure where to place it.

**Comment by paskal** [comment]
That is expected behavior currently; that's how the frontend knows that the user is not known to the backend. That behavior might be changed soon, as discussed in https://github.com/umputun/remark42/issues/1188. The backend already supports such checks with status code 200.
